<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryManager</class>
 <widget class="QDialog" name="HistoryManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>History</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="import_pushButton">
           <property name="text">
            <string>Import</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="delete_toolButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="list-remove">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="clear_toolButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="edit-clear">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="filesListWidget">
         <property name="currentRow">
          <number>0</number>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Test</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>test</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>test</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWebView" name="itemsWebView">
         <property name="font">
          <font>
           <family>Sans Serif</family>
          </font>
         </property>
         <property name="url">
          <url>
           <string>file:///home/flareguner/Development/projects/qphoenix/src/historymanager/example.html</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HistoryManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HistoryManager</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
